packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -274,7 +274,9 @@ async function generateBase64({ file, args = {}, reporter }) {
   })
   let pipeline
   try {
-    pipeline = sharp(file.absolutePath)
+    pipeline = !options.failOnError
+      ? sharp(file, { failOnError: false })
+      : sharp(file)
```

Comment:
```suggestion
      ? sharp(file.absolutePath, { failOnError: false })
      : sharp(file.absolutePath)
```
This should be fiel `absolutePath` (should also fix tests)